ocaml/idl/markdown_backend.ml

```diff
@@ -154,8 +154,8 @@ let print_field_table_of_obj printer x =
   if x.contents=[] then
     printer ("Class "^(escape x.name)^" has no fields.")
   else begin
-    printer "|Field|Type|Qualifier|Description|";
-    printer "|:---|:---|:---|:---|";
+    printer "|Field            |Type               |Qualifier  |Description                         |";
```

Comment:
The lines here are extended to make the generate md more readable, am I correct?